Logix/Avatar/AvatarRootSlot.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using FrooxEngine;
+using FrooxEngine.LogiX;
+using FrooxEngine.LogiX.Math;
+using BaseX;
+using FrooxEngine.CommonAvatar;
+
+
+namespace FrooxEngine.LogiX.Avatar
+{
+    [Category("LogiX/Avatar")]
+    [NodeName("Avatar Root Slot")]
+    public class AvatarRootSlot : LogixOperator<Slot>
+    {
+        public readonly Input<User> User;
+        public override Slot Content
+        {
+            get
+            {
+                User user = User.Evaluate();
+                if (user == null)
+                    return null;
+                Slot slot = user.Root.Slot;
+                foreach (var item in slot.GetAllChildren())
```

Comment:
Slot.GetComponentsInChildren<AvatarRoot> ?

Reply:
GetFirstDirectComponentsInChildren probably actually, that stops the search when it finds one.

Reply:
Resolving this as they seemed to have done what you said, nice to see you helping ^_^